src/index.js

```diff
@@ -1,2 +1,53 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+const BUTTON_NUMBERS = [1, 2, 3];
+const INITIAL_STATE = { count: 0 };
+
+function render({ count }) {
+  const handleClickMeClick = (prevCount) => {
+    render({ count: prevCount + 1 });
+  };
+
+  const handleNumberButtonClick = (number) => {
+    render({ count: number });
+  };
+
+  const element = (
+    <div>
+      <p>
+        <button type="button" onClick={() => handleClickMeClick(count)}>
+          Click me! (
+          {count}
+          )
+        </button>
+      </p>
+      <p>{BUTTON_NUMBERS.map((number) => (<button type="button" onClick={() => handleNumberButtonClick(number)}>{number}</button>))}</p>
```

Comment:
```suggestion
      <p>{BUTTON_NUMBERS.map((number) => (
        <button type="button" onClick={() => handleNumberButtonClick(number)}>
          {number}
        </button>
        )}
      </p>
```

이런식으로 코드가 길어질 때 적절히 줄바꿈하여 정리해주시면 좋아요 :)